spec/features/resident_can_locate_problem_spec.rb

```diff
@@ -90,4 +90,24 @@
 
     expect(page).to have_content('N1 6NU')
   end
+
+  scenario 'performing a search, but not selecting any address' do
+    matching_property = {
+      'property_reference' => 'abc123',
+      'short_address' => 'Flat 1, 8 Hoxton Square, N1 6NU',
+    }
+
+    fake_api = instance_double(JsonApi)
+    allow(fake_api).to receive(:get).with('properties?postcode=N1 6NU').and_return([matching_property])
+    allow(fake_api).to receive(:get).with('properties/').and_return({})
```

Comment:
Is this line required? The test seems to pass without it.

Reply:
Good spot. I think this was left over from some earlier testing :+1: